### DISCUSSION TITLE
There is not enough space on the disk error in windows

### DISCUSSION THREAD

**Discussion by kamesh-git** [community/community#67105] [Programming Help]
### Body

I had a linux operationg system and an windows OS in same machine via dual boot and for linux OS i made a seperate partition of 200GB from my from 1TB hard disk. Few weeks ago i deleted my 200GB linux partiton via disk management in windows and the 200GB space become unallocated but i can't able to merge or extend the partition the error it shows is 
there is not enough available space on the disk 

![`there is not enough available space on the disk`](https://i.stack.imgur.com/gA6LQ.png)

i opened the diskpart in cmd and tried to list volumes and partition in that but i couldn't able to view my unallocated partition

![`diskpart`](https://i.stack.imgur.com/LuEiL.png)

![`unallocated partition`](https://i.stack.imgur.com/O1f04.png)

kindly help to resolve this issue i want to either extend my partition or make it a new simple volume

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hafiz-muhammad** [comment]
You can contact Microsoft Windows support here: [https://support.microsoft.com/en-us/contactus](https://support.microsoft.com/en-us/contactus)
